source/locales.tex

```diff
@@ -23,7 +23,7 @@
 \ref{locales} & Locales                   &   \tcode{<locale>}    \\
 \ref{locale.categories} & Standard \tcode{locale} categories &   \\ \rowsep
 \ref{c.locales} & C library locales       &   \tcode{<clocale>}   \\ \rowsep
-\ref{text.encoding} & Text encodings identification & \tcode{<ctext.encoding>}   \\
+\ref{text.encoding} & Text encodings identification & \tcode{<ctext_encoding>}   \\
```

Comment:
Header name is still wrong.

Reply:
This was fixed in a FIXUP:
```
git show 871857420420f07c3f4f026bb316058e1bbc0d69 | grep "Text encodings identification"
-\ref{text.encoding} & Text encodings identification & \tcode{<ctext.encoding>}   \\
+\ref{text.encoding} & Text encodings identification & \tcode{<ctext_encoding>}   \\
```

Resolving, please reopen if you're not seeing this change.

Reply:
Sorry, but the spelling of the header name is still `<ctext_encoding>` (note leading `c`), which is wrong.

Reply:
Oh! Thanks for clarifying :). fixed